portfolio.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>My Portfolio</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>	
+	
+	<body>
+		
+		<nav class="nav_bar">
+				<p>Karen Schulz</p>
+			
+			<div>
+				<a href="index.html">Home</a>
+				<a href="contact.html">Contact</a>
+			</div>
+			
+		</nav>
+		
+	
+		<div class="portfolio_page">
+			
+			<img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture in the winter, mustard bobble hat and blue scarf">
+		</div>
+			
+		<article class="content">
+			
+			<h1>Welcome to my Portfolio</h1>
+			
+			<div class="portfolio1">
+					<img class="first_work" src="images/green_back.png" alt="my_first_work_hamster_illustration_green_background">
+				
+				<p>I love to do illustrations, check my creations</p>
+				
+			</div>
+			
+		<div class="my_potfolio">
+				
+			<div class="portfolio2">
+					<img class="second_work" src="images/eating.png" alt="my second work hasmter eating a seed, fucsia background">
+				<p>I love html and CSS</p>
+			</div>
+				
+			<div class="portfolio3">
+					<img class="third_work" src="images/full-pouch.png" alt="third illustration with hamster full pouch, yellow background">
+					
+				<p>I've written two children's books</p>
+			</div>
+		</div>
+				
+		<div class="links">
+				<a href="index.html">Home</a>
+				<a href="contact.html">Contact</a>
+		</div>
+		
+		</article>
+		
+		<footer>&copy Copyright. All rights reserved. Karen Schulz, January 2021</footer>
```

Comment:
: after &copy

Reply:
OK... it works with semicolon.

Reply:
At least with me, it doesn't work with : 
👍

Reply:
Problem solved